src/gql/campaigns.ts

```diff
@@ -54,6 +54,9 @@ export type AdminCampaignResponse = BaseCampaignResponse & {
   coordinator: {
     person: { firstName: string; lastName: string }
   }
+  incomingTransfers: [{ amount: number }]
+  outgoingTransfers: [{ amount: number }]
```

Comment:
Most probably this needs to be:

```suggestion
  incomingTransfers: { amount: number }[]
  outgoingTransfers: { amount: number }[]
```

In the campaign entity we can see:

https://github.com/podkrepi-bg/api/blob/90cb4c5e1a54208f2be181fc6088e846a2ff45e1/apps/api/src/domain/generated/campaign/entities/campaign.entity.ts#L36-L37